docs/dev_guide/notebooks/backend.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<ul><li>I would rather it say "plan is to remove  the<code>numpy</code> backend since code that relies on the numpy backend instead of JAX is not automatically tested for correctness by the GitHub CI."</li><li>I think we can still do finite difference with JAX? just not by default. maybe don't include  "for those who wat to compare the results with finite difference".</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>